javascript/htmlContent.js

```diff
@@ -78,7 +78,7 @@ export const switchTitle = version => {
     </div>
     <div class="title-text-OTHEREDITIONS">
       <span class="title-text-OTHEREDITIONS">
-    <a href="https://source-academy.github.io/interactive-sicp">Back to Interactive SICP</a></span>
+    <a href="https://source-academy.github.io/sicpjs">Back to Interactive SICP</a></span>
```

Comment:
This is the link from the comparison edition to SICP JS. Should we use the new version?

I think we should avoid linking to github.io when it comes to the textbook.

Reply:
Was sloppy here, sorry about that. I'll make the necessary changes.